### PR TITLE
docs(ui5-label): enhance the documentation and the samples

### PR DESCRIPTION
**Pull request by LidiyaGeorgieva**
The correct usage of ui5-label is to be connected to some input element (using for attribute). This is documented in all the test pages.
Also in the Playgrounds pages for ui5-label are added corresponding inputs.

Related to: #6488

### CODE REVIEW

File: packages/main/src/Label.ts [hunk 1]
```diff
@@ -17,8 +17,8 @@ import labelCss from "./generated/themes/Label.css.js";
  *
  * The <code>ui5-label</code> is a component used to represent a label,
```
Comment (TeodorTaushanov) [inline packages/main/src/Label.ts:18]:
```
 * The <code>ui5-label</code> is a component used to represent a label for elements like input, textarea, select. <br><br>
The <code>for</code> property of the <code>ui5-label</code> must be the same as the id attribute of the related input element.<br><br>
Screen readers read out the label, when the user focuses the labelled control.
```

**Review by TeodorTaushanov — DISMISSED** [review]

File: packages/playground/_stories/main/Label/Label.stories.ts [hunk 2]
```diff
@@ -37,37 +38,63 @@ const Template: UI5StoryArgs<Label, StoryArgsSlots> = (args) => {
...
 export const Basic = Template.bind({});
 Basic.args = {
-	default: "The quick brown fox jumps over the lazy dog"
+	showColon: true,
 };
+Basic.decorators = [
+	(story, { args }) => {
+		return html`
+	${story({ args: { ...args, for: "myInputSimple", default: "Simple Label"}})}
+	<ui5-input id="myInputSimple"></ui5-input>
```
Comment (gmkv) [inline packages/playground/_stories/main/Label/Label.stories.ts:47]: Extract this input into a function and use it in all places. Like the `SetWidth200Px` decorator.

**Review by gmkv — CHANGES_REQUESTED** [review]

File: packages/playground/_stories/main/Label/Label.stories.ts [hunk 3]
```diff
@@ -29,45 +29,85 @@ const Template: UI5StoryArgs<Label, StoryArgsSlots> = (args) => {
...
-	default: "Required label with colon",
 	showColon: true,
 	required: true,
+	default: "Required label with colon",
 };
+RequiredWithColon.decorators = [
+	(story, { args }) => {
+		return html`
+	${story({ args: { ...args, for: "myInputRequired"}})}
+`;
```
Comment (gmkv) [inline packages/playground/_stories/main/Label/Label.stories.ts:68]: these can be deleted now

File: packages/playground/_stories/main/Label/Label.stories.ts [hunk 4]
```diff
@@ -29,45 +29,85 @@ const Template: UI5StoryArgs<Label, StoryArgsSlots> = (args) => {
...
 >
 	${unsafeHTML(args.default)}
 </ui5-label>`;
 };
 
+const addInput = (id: string) => {
+	return (story: () => unknown) => html`
+${story()}
+<ui5-input id="${id}"></ui5-input>`;	
+};
```
Comment (gmkv) [inline packages/playground/_stories/main/Label/Label.stories.ts:42]:
```ts
const addInput = (id: string): Decorator => {
	return (story, { args }) => {
		return html`
${story({ args: { ...args, for: id } })}
<ui5-input id="${id}"></ui5-input>`;
	};
};
```
the decorator import is from `import type { Decorator } from "@storybook/web-components"`

File: packages/playground/_stories/main/Label/Label.stories.ts [hunk 5]
```diff
@@ -29,45 +29,85 @@ const Template: UI5StoryArgs<Label, StoryArgsSlots> = (args) => {
...
 	default: "Long labels are truncated by default.",
 };
-Truncating.decorators = [SetWidth200Px];
+Truncating.decorators = [
+	SetWidth200Px,
+	(story, { args }) => {
+		return html`
+	${story({ args: { ...args, for: "myInputTruncation", className: "limitedWidth" }})}
+`;
+},
```
Comment (gmkv) [inline packages/playground/_stories/main/Label/Label.stories.ts:92]: rewrite the SetWidth200Px as:
```ts
const SetWidth200Px: Decorator = (story, { args }) => {
	return html`
<style>
	.w200 {
		width: 200px;
	}
</style>
${story({ args: { ...args, className: "w200" } })}`;
};
```
and this middle function can be deleted

File: packages/playground/_stories/main/Label/Label.stories.ts [hunk 6]
```diff
@@ -29,45 +29,85 @@ const Template: UI5StoryArgs<Label, StoryArgsSlots> = (args) => {
 	?required="${ifDefined(args.required)}"
 	?show-colon="${ifDefined(args.showColon)}"
 	wrapping-type="${ifDefined(args.wrappingType)}"
+	class="${ifDefined(args.className)}"
 >
 	${unsafeHTML(args.default)}
 </ui5-label>`;
```
Comment (gmkv) [inline packages/playground/_stories/main/Label/Label.stories.ts:35]: please remove the newlines here:
```ts
>${unsafeHTML(args.default)}</ui5-label>`;
```
it causes some extra whitespace that #7042 describes

File: packages/main/test/pages/Label.html [hunk 7]
```diff
@@ -24,6 +24,7 @@
 </head>
 
 <body class="label1auto">
+	<div id="message"><ui5-message-strip design="Information">The proper use of the component is to define a label for elements like input, textarea, select.</ui5-message-strip></div>
```
Comment (gmkv) [inline packages/main/test/pages/Label.html:27]: No-one will read these pages. These test files are not part of our documentation

File: packages/playground/_stories/main/Label/Label.stories.ts (hunk 6 above)
Comment (gmkv) [reply packages/playground/_stories/main/Label/Label.stories.ts]: All on one line

File: packages/playground/_stories/main/Label/Label.stories.ts [hunk 8]
```diff
@@ -29,45 +30,70 @@ const Template: UI5StoryArgs<Label, StoryArgsSlots> = (args) => {
...
-
-${story()}`;
+${story({ args: { ...args, className: "w200" } })}`;
+};
 
 export const Truncating = Template.bind({});
 Truncating.args = {
+	showColon: true,
 	default: "Long labels are truncated by default.",
+	className: "limitedWidth",
```
Comment (gmkv) [inline packages/playground/_stories/main/Label/Label.stories.ts:80]: not needed any more

**Review by gmkv — CHANGES_REQUESTED** [review]

**Review by gmkv — APPROVED** [review]

**Pull request merged**